data/hai/hai missions.txt

```diff
@@ -1557,3 +1557,71 @@ mission "Strider notification"
+	on offer
+		event "Hai Translator Ban Lifted" 20 30
+		fail
+
+event "Hai Translator Ban Lifted"
+
+
+
+mission "Hai Translator Ban Lifted"
+	invisible
+	landing
+	source
+		government Hai
+	to offer
+		has "event: Hai Translator Ban Lifted"
+	on offer
+		conversation
+			`After landing, you see several humans and Hai talking to one another, each in their native language, but they all seem to understand one another. Walking around the spaceport for a while, this seems to be ubiquitous: all humans can understand the language of the Hai. A Hai in an official-looking uniform comes up to you.`
+			`	"Excuse me, would you like a Hai-human translator?"`
+			label decision
+			choice
+				`	"Why are these suddenly available now?"`
+					goto why
+					to display
+						not "Hai Translator Ban Lifted: why?"
+				`	"How did humans never notice this?"`
+					goto how
+					to display
+						not "Hai Translator Ban Lifted: how?"
+						has "Hai Translator Ban Lifted: why?"
+				`	"I already have one."`
+					decline
+					to display
+						has "language: Hai"
+				`	"Yes, that would be great."`
+					goto yup
+					to display
+						not "language: Hai"
+				`	"Certainly not. I prefer the mystery of not knowing what the Hai are saying."`
+					decline
+			label why
+			action
+				set "Hai Translator Ban Lifted: why?"
+			`	"They were illegal until now. Humans and Hai created translation devices weeks after first contact; it is an ancient technology to both races. Our Elders felt a language barrier would facilitate Hai secrecy, and human authorities agreed, so they banned the devices.`
+			`	"Very few Hai actually know the human language. We've been using these devices." The official lifts fur along the side of their head, and you can see a device attached to their ear, the front of their neck, and the back of their head. "The ones we wear only work for Hai, but we have ones designed for humans`
+			`	"Times have changed, and we must shatter the language barrier to build friendships. Do you want a translator?"`
+				goto decision
+			label how
+			action
+				set "Hai Translator Ban Lifted: how?"
+			`	The official smiles. "Your species is resilient, forgiving, and tolerant, but not observant. How did humans never notice the Hai, just six jumps away, for a century? How did the Pug hide right next to you? We have high hopes for your skills at peacemaking and building friendships, and those are the most important traits of a neighbor.`
+			`	"Do you want a translator?"`
+				goto decision
+			label yup
+			action
+				set "language: Hai"
+			`	The official hands you a device that you strap around your head and connect to your ear. The interface is quite intuitive, and it quickly translates between human and Hai languages, both text and speech. There's a standard interface port; when you connect to your datapad, it downloads a translation program. When you look in a mirror, you notice the device is difficult to see, possibly due to a holographic camouflage.`
```

Comment:
You read it backwards. The "it" is ambiguous. The translation device transferred its program to the datapad. This way you could, for example, translate an entire database of Hai text for automated analysis, rather than having to read through it page by page.